ui/panels-plugin/src/plugins/stat-chart/StatChartOptionsEditorSettings.test.tsx

```diff
@@ -66,12 +66,12 @@ describe('StatChartOptionsEditorSettings', () => {
     const calcSelector = screen.getByRole('combobox', { name: 'Calculation' });
     userEvent.click(calcSelector);
     const firstOption = screen.getByRole('option', {
-      name: /First/,
+      name: /First number/,
     });
     userEvent.click(firstOption);
     expect(onChange).toHaveBeenCalledWith(
       expect.objectContaining({
-        calculation: 'First',
+        calculation: 'FirstNumber',
```

Comment:
Switching a few tests to look for `FirstNumber` instead of `First` because it's more specific and doesn't run into RTL errors where it finds two matches.